views/layouts/authenticated-layout.pug

```diff
@@ -22,7 +22,7 @@ html
       form.navbar__form.navbar--left(method='get', action='/items')
         .form.form--inline
           input.form__control.navbar__searchbox(type='text', name='search', placeholder='Search')
-      .navbar__star.btn.btn--star
+      div.navbar__star.btn.btn--star#star-toggle
```

Comment:
We can remove the `div` tag here as pug implicitly creates a div when no tag name is specified.